omegatrade-app/backend/app/controller/user.controller.js

```diff
@@ -112,33 +74,28 @@ exports.login = function (req, res) {
  * 
  * @param {*} req contains request headers and payload businessEmail,password
  */
-exports.getToken = function (req, res) {
-    const body = req.body;
-    if (body && body.provider === 'GOOGLE') {
-        User.findUser(body.email, function (err, user) {
+exports.getToken = async function (req, res) {
+    try {
+        const body = req.body;
+        if (body && body.provider === 'GOOGLE') {
+            const [user] = await User.findUser(body.email)
             if (user && user.userId) {
```

Comment:
I would have expected this if statement to check whether the provider of the user is `Google` (or empty).